### PR TITLE
Telegram & discord bot

### PR DESCRIPTION
**Pull request by haythem96**
Telegram bot done, `Crab OTC Testnet` and `Crab OTC` on telegram

### CODE REVIEW

**Comment by haythem96** [comment]
Posting here full env needed
```
NEXT_PUBLIC_INFURA_API_KEY=
NEXT_PUBLIC_PORTAL_API_KEY=
NEXT_PUBLIC_CHAIN_ID=
NEXT_PUBLIC_ENABLE_ANALYTICS=
TELEGRAM_BOT_TOKEN=
TELEGRAM_CHANNEL_ID=
IPFS_PROJECT_ID=
IPFS_PROJECT_SECRET=
FIREBASE_PRIVATE_KEY=
FIREBASE_CLIENT_EMAIL=
DISCORD_BOT_TOKEN=
DISCORD_CHANNEL_ID=
```

File: pages/crab-otc/[id].tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { BIG_ZERO } from '../../constants/numbers'
 import { Expiry } from '../../container/CrabOTC/Expiry'
 import ApprovalsOtc from '../../container/CrabV2/Auction/ApprovalsOtc'
 import { OTCInfo } from '../../container/CrabV2/CrabOTC/OTCInfo'
-import { Nav } from '../components/navbars/Nav'
+import { Nav } from '../../components/navbars/Nav'
```
Comment (sanandnarayan) [inline pages/crab-otc/[id].tsx:16]: @haythem96 whats this line? I mean what does Nav have to do with this PR?

File: pages/api/crabotc/deleteOTC.ts [hunk 2]
```diff
@@ -16,6 +17,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   }
 
   await deleteOtc(crabOtc)
+  sendTelegramMessage(crabOtc, crabOtc.data, true)
```
Comment (sanandnarayan) [inline pages/api/crabotc/deleteOTC.ts:20]: do we want to send a msg when one deletes OTC?

File: server/utils/discord-bot.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+// When the client is ready, run this code (only once)
+client.once('ready', () => {
+	console.log('Ready!');
+});
+
+// Login to Discord with your client's token
+client.login(process.env.DISCORD_BOT_TOKEN);
+
+export const sendDiscordMessage = () => {
+    client.channels.fetch(String(process.env.DISCORD_CHANNEL_ID))
```
Comment (sanandnarayan) [inline server/utils/discord-bot.ts:16]: This is WIP ?

File: server/utils/telegram-bot.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        notificationType = 'Updated OTC'
+    }
+    else {
+        notificationType = 'New OTC'
+    }
+
+    let msg: string;
+    if (crabOTCData.depositAmount !== 0) {
+        notificationType += ' deposit'
+        msg = `${notificationType}: Created by: ${crabOTCData.createdBy} Deposit amount: ${crabOTCData.depositAmount} Quantity: ${crabOTCData.quantity} Min price per oSQTH: ${crabOTCData.limitPrice} Expiry: ${crabOTCData.expiry}`
```
Comment (sanandnarayan) [inline server/utils/telegram-bot.ts:24]: lets change this messaging to

Bid for 54.39 oSQTH limit price: 0.01 link
Offer 45.29 oSQTH limit price: 0.01 link

**Pull request merged**